server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -201,8 +198,7 @@ public CompletableFuture<PushResultDto> push(
             Author author,
             CommitMessageDto commitMessage,
             @RequestConverter(ChangesRequestConverter.class) Iterable<Change<?>> changes) {
-        final User user = AuthUtil.currentUser(ctx);
-        checkPush(repository.name(), changes, user.isSystemAdmin());
+        checkMetaRepoPush(repository.name(), changes);
```

Comment:
Should we remove this validation as accessing `meta` is prohibited by the decorator?

Reply:
A system admin can still do the push and I left it because of these lines:
https://github.com/line/centraldogma/pull/1115/files#diff-365d4212ade9fe684cd89a04b979999f32b398703bd3ebbfa99d51d10617c3b4R455